<ui version="4.0" >
 <author>Dane Springmeyer</author>
 <class>DrawGear</class>
 <widget class="QDialog" name="DrawGear" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Next Polygon</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QWidget" native="1" name="widget_2" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="type_label" >
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="penny_label_3" >
          <property name="text" >
           <string>Weighting (pennies):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_3" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pl_label_4" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Check which types of species are caught in this area:</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="s5" >
          <property name="text" >
           <string>Benthic fish</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s2" >
          <property name="text" >
           <string>Coastal reef fish</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s3" >
          <property name="text" >
           <string>Deep reef fish</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="s4" >
          <property name="text" >
           <string>Migratory fish</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s1" >
          <property name="text" >
           <string>Sharks and Skates</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s6" >
          <property name="text" >
           <string>Shrimp</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="s7" >
          <property name="text" >
           <string>Other</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>196</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbnShapeFinished" >
        <property name="text" >
         <string>Finished With Shapes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnShapeDiscard" >
        <property name="text" >
         <string>Discard Last Shape...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnMoreShapes" >
        <property name="text" >
         <string>Draw More Shapes...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
